tcl/org.testeditor.tcl.dsl.ui/src/org/testeditor/tcl/dsl/ui/editor/TclModelDragAndDropUpdater.xtend

```diff
@@ -182,7 +185,8 @@ class TclModelDragAndDropUpdater {
 	 * the modification of the import section must take place before actually adding the new
 	 * test step component.
 	 */
-	public def void updateImports(TclModel tclModel, Component droppedObject, String qualifiedName) {
+	public def void updateImports(TclModel tclModel, EObject dropTarget, Component droppedObject,
+		String qualifiedName) {
```

Comment:
I can't see to which object the `qualifiedName` belongs. Why do we need to pass this as a parameter anyway? We can use the qualifiedNameProvider to retrieve it.